plugins/rtmp-services/data/package.json

```diff
@@ -1,11 +1,11 @@
 {
     "$schema": "schema/package-schema.json",
     "url": "https://obsproject.com/obs2_update/rtmp-services/v5",
-    "version": 260,
+    "version": 261,
     "files": [
         {
             "name": "services.json",
-            "version": 260
+            "version": 261
         }
     ]
-}
+}
```

Comment:
Revert this change. Always end the file with a newline.

Reply:
Fixed!